openspace.cfg

```diff
@@ -60,6 +60,8 @@ Asset = "default"
 -- Asset = "apollo_sites"
 -- Asset = "touch"
 -- Asset = "dawn"
+-- Asset = "mas"
+Asset = "bastille-mas-model"
```

Comment:
we would want new scenes to be added here, but we still want default to be the default.